modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/data_renderers/ImageRenderer.js

```diff
@@ -19,53 +19,31 @@ import React from 'react';
 import {getValueFromItem} from '../utilities/index';
 
 function ImageRenderer(props) {
+	const imgProps =
```

Comment:
Here's an abbreviation that escaped @wincent's keen 👀

Reply:
Maybe this one is so common that we don't mind it though

Reply:
[No exceptions](https://en.wikipedia.org/wiki/Slippery_slope)!

Except, of course, for [the exceptions](https://github.com/liferay/liferay-frontend-guidelines/blob/master/general/naming.md#examples-of-good-abbreviations).